pkg/workload/base.go

```diff
@@ -11,13 +11,23 @@ import (
 
 // TpcState saves state for each thread
 type TpcState struct {
+	DB   *sql.DB
 	Conn *sql.Conn
 
 	R *rand.Rand
 
 	Buf *util.BufAllocator
 }
 
+func (t *TpcState) RereshConn(ctx context.Context) error {
```

Comment:
`RefreshConn`

Reply:
Good catch, I feel weird until now I got the reason...